google/cloud/spanner_v1/snapshot.py

```diff
@@ -243,7 +249,13 @@ def execute_sql(
             timeout=timeout,
         )
 
-        iterator = _restart_on_unavailable(restart)
+        trace_attributes = {"db.statement": sql}
```

Comment:
So we just directly expose `sql` to traces without any processing, right? We probably should document this somewhere to let users know if they have PII data or data privacy concerns. 

I wonder why do we need this `db.statement`? How do we come up with the decision to add this field?

Reply:
`db.statement` is mentioned as a "required" attribute in the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#call-level-attributes) (although we can do whatever we want). We could sanitize the data for PII ourselves, although that's probably a bad idea. Personally I'm fine with removing it but I think it would be a very useful attribute for debugging. And correct me if I'm wrong, but if done right the SQL should be split into the query (statement) and actual values (params) right? And since we only upload the query it could be fine?

Reply:
I think I am fine with it. 

You're right. It is supposed to split query and params. However, it might happen that users just compose a query string and pass an empty params. Not all users will follow the best practice.